index.html

```diff
@@ -95,13 +95,60 @@ <h1 class="logo__text">Renome</h1>
         </header>
         <main>
             <div class="content">
-                <img src="/images/carousel_img_1.png"/>
-                <img src="/images/carousel_img_2.png"/>
-                <img src="/images/carousel_img_3.png"/>
-            </div>
+                <div class="carousel" id="carousel">
+                    <div class="carousel__text" id="carousel_text">
+                        <div class="carousel__title">made with love</div>
+                        <div class="carousel__subtitle">for you</div> 
+                    </div>
+                    <div id="carousel_box">
+                        <object class="carousel__object" data="/images/carousel_img_1.png" type="image/png"></object>
```

Comment:
At first I misunderstood the requirement "Hero carousel should be developed using objects, each slide should be an object". And then I did not change because it might to use HTML <object> tag for embedding images.

Reply:
Changed to img tag